tests/integration/utils.ts

```diff
@@ -133,3 +135,20 @@ export const debug = async (page: Page) => {
     stdin.on('data', onKeyPress);
   });
 };
+
+export const selectTestNet = async (wallet: WalletPage) => {
+  await wallet.clickSettingsButton();
+  await wallet.page.click(createTestSelector(SettingsSelectors.ChangeNetworkAction));
+  await wallet.page.waitForTimeout(1200);
+
+  const networkListItems = await wallet.page.$$(
+    createTestSelector(SettingsSelectors.NetworkListItem)
+  );
+  console.log(networkListItems[1].innerHTML());
+  await networkListItems[1].click();
+};
+
+export const timeDifference = (startDate: Date, endDate: Date) => {
```

Comment:
Take a look at https://developer.mozilla.org/en-US/docs/Web/API/Performance/now , might be able to use this here